### PR TITLE
Update sync_gokb_packages.groovy

### PR DESCRIPTION
**Pull request by philboeselager**

### CODE REVIEW

File: scripts/sync_gokb_packages.groovy [hunk 1]
```diff
@@ -39,7 +39,7 @@ while ( moredata ) {
         data.TIPPs.TIPP.each { xmltipp ->
 
           // TIPP.
-          def newtipp = directAddFields (xmltipp, ['medium', 'url', 'series', 'subjectArea'], addCoreItems ( xmltipp ))
+          def newtipp = directAddFields (xmltipp, ['medium', 'url', 'series', 'subjectArea', 'status', 'name', 'lastUpdated', 'publisherName', 'dateFirstInPrint', 'dateFirstOnline', 'publicationType', 'firstAuthor', 'volumeNumber', 'editionStatement', 'firstEditor', 'parentPublicationTitleId', 'precedingPublicationTitleId', 'lastChangedExternal', 'importId'], addCoreItems ( xmltipp ))
```
Comment (hornmo) [inline scripts/sync_gokb_packages.groovy]: `status` & `name` are handled via `addCoreItems`, `lastUpdated` is immutableand can be disregarded.

File: scripts/sync_gokb_packages.groovy [hunk 2]
```diff
@@ -59,23 +59,21 @@ while ( moredata ) {
...
+            if (!type || type == 'JournalInstance' || type == 'Serial') {
+                newtipp.title.type = 'Serial'
+            }
+            else if (type == 'DatabaseInstance') {
+                newtipp.title.type = 'Database'
+            }
+            else if (type == 'BookInstance'|| type == 'Monograph') {
+                newtipp.title.type = 'Book'
+            }
+            newtipp.platform = directAddFields (xmltipp.platform, ['primaryUrl'], addCoreItems ( xmltipp.platform ))
```
Comment (hornmo) [inline scripts/sync_gokb_packages.groovy]: The TIPP platform is not depending on title information.

**Review by hornmo — CHANGES_REQUESTED** [review]

File: scripts/sync_gokb_packages.groovy (hunk 1 above)
Comment (philboeselager) [reply scripts/sync_gokb_packages.groovy]: Done.

File: scripts/sync_gokb_packages.groovy (hunk 2 above)
Comment (philboeselager) [reply scripts/sync_gokb_packages.groovy]: Moved line 74 out of the title check.

File: scripts/sync_gokb_packages.groovy [hunk 3]
```diff
@@ -71,9 +71,9 @@ while ( moredata ) {
             else if (type == 'BookInstance'|| type == 'Monograph') {
                 newtipp.title.type = 'Book'
             }
-            newtipp.platform = directAddFields (xmltipp.platform, ['primaryUrl'], addCoreItems ( xmltipp.platform ))
             resourceFieldMap['tipps'].add(newtipp)
```
Comment (hornmo) [inline scripts/sync_gokb_packages.groovy]: This line adds the finished TIPP to the result and needs to be at the very end.

**Review by hornmo — CHANGES_REQUESTED** [review]

File: scripts/sync_gokb_packages.groovy (hunk 3 above)
Comment (philboeselager) [reply scripts/sync_gokb_packages.groovy]: Done.

**Review by hornmo — APPROVED** [review]

**Pull request merged**